yetanotherco/aligned_layer
# feat(explorer):  batch-size chart

# Add batch-size chart to the explorer 

## Description

Based off #1635 (shout out @MarcosNicolau) adds a chart to the explorer that shows the batch size of batches over the last 100 blocks.

## To Test

Start a local devnet.
Run the explorer.
Send proofs at various rates and observe the batch size increase or decrease.

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
